.github/workflows/main-build.yml

```
@@ -36,7 +36,7 @@ jobs:
             type=ref,event=branch,pattern=latest
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v3
+        uses: docker/login-action@v4
```

v4 of `docker/login-action` doesn't exist: https://github.com/docker/login-action/releases

```suggestion
        uses: docker/login-action@v3
```